pages/speakers.js

```diff
@@ -0,0 +1,30 @@
+import Page from '../components/Page'
+import speakerEvents from '../static/speakers.json'
+
+export default () => {
+
+  const speakerList = [];
+
+  speakerEvents.map((event, id) => {
```

Comment:
Map returns an array, and it makes the code simpler by using `.map` and `.filter` 😄